src/week1.fibonacci/lunar.js

```diff
@@ -0,0 +1,97 @@
+// > 출처: https://programmers.co.kr/learn/courses/30/lessons/12945 (프로그래머스, 난이도 2/5)
+
+function solution(n) {
+  if (n < 2) return;
```

Comment:
이전 시도에서 사용한 `return n`이 들어가면 완벽할 것 같아요!
```suggestion
  if (n < 2) return n;
```